js/ace.js

```diff
@@ -0,0 +1,1021 @@
+            'close': this.safeString (ticker, 'last_price'),
+            'last': this.safeString (ticker, 'last_price'),
+            'previousClose': undefined,
+            'change': undefined,
+            'percentage': undefined,
+            'average': undefined,
+            'baseVolume': this.safeString (ticker, 'base_volume'),
+            'quoteVolume': this.safeString (ticker, 'quote_volume'),
+            'info': ticker,
+        }, market);
+    }
+
+    async fetchTicker (symbol, params = {}) {
+        /**
+         * @method
+         * @name ace#fetchTicker
+         * @description fetches a price ticker, a statistical calculation with the information calculated over the past 24 hours for a specific market
+         * @param {string} symbol unified symbol of the market to fetch the ticker for
+         * @param {object} params extra parameters specific to the ace api endpoint
+         * @returns {object} a [ticker structure]{@link https://docs.ccxt.com/en/latest/manual.html#ticker-structure}
+         */
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const response = await this.publicGetOapiV2ListTradePrice (params);
+        const marketId = market['id'];
+        const ticker = this.safeValue (response, marketId, {});
+        //
+        //     {
+        //         "BTC/USDT":{
+        //             "base_volume":229196.34035399999,
+        //             "last_price":11881.06,
+        //             "quote_volume":19.2909
+        //         }
+        //     }
+        //
+        ticker['id'] = marketId;
+        return this.parseTicker (ticker, market);
+    }
+
+    async fetchTickers (symbols = undefined, params = {}) {
+        /**
+         * @method
+         * @name ace#fetchTickers
+         * @description fetches price tickers for multiple markets, statistical calculations with the information calculated over the past 24 hours each market
+         * @param {[string]|undefined} symbols unified symbols of the markets to fetch the ticker for, all market tickers are returned if not assigned
+         * @param {object} params extra parameters specific to the ace api endpoint
+         * @returns {object} an array of [ticker structures]{@link https://docs.ccxt.com/en/latest/manual.html#ticker-structure}
+         */
+        await this.loadMarkets ();
+        const response = await this.publicGetOapiV2ListTradePrice ();
+        //
+        //     {
+        //         "BTC/USDT":{
+        //             "base_volume":229196.34035399999,
+        //             "last_price":11881.06,
+        //             "quote_volume":19.2909
+        //         }
+        //     }
+        //
+        const tickers = [];
+        const pairs = Object.keys (response);
+        for (let i = 0; i < pairs.length; i++) {
+            const marketId = pairs[i];
+            const ticker = this.safeValue (response, marketId);
+            ticker['id'] = marketId;
```

Comment:
same here, since we already have a loop you can manually call `parseTicker` providing the market and end with `return this.filterByArray (results, 'symbol', symbols);`